From d5799e54b2dc52d7f2abb70c1b01f16e40edb989 Mon Sep 17 00:00:00 2001
From: Surface Duo Builder <bariccatti@gmail.com>
Date: Sun, 19 Apr 2026 19:52:38 +0000
Subject: [PATCH] Remove Samsung light HIDL deps from frameworks/base

vendor/interfaces no longer ships Samsung HIDL stubs. Remove
vendor.samsung.hardware.light@2.0 and @3.0 deps from services/core/jni.
---
 services/core/jni/Android.bp | 2 --
 1 file changed, 2 deletions(-)

diff --git a/services/core/jni/Android.bp b/services/core/jni/Android.bp
index 2a1203e45..6bd4e821b 100644
--- a/services/core/jni/Android.bp
+++ b/services/core/jni/Android.bp
@@ -205,8 +205,6 @@ cc_defaults {
         "android_location_flags_c_lib",
         "server_configurable_flags",
         "service.incremental",
-        "vendor.samsung.hardware.light@2.0",
-        "vendor.samsung.hardware.light@3.0",
     ],
 
     static_libs: [
-- 
2.34.1

